src/helpers/xml-builder.js

```diff
@@ -727,12 +739,16 @@ const buildRunOrRuns = async (vNode, attributes, docxDocumentInstance) => {
 
     for (let index = 0; index < vNode.children.length; index++) {
       const childVNode = vNode.children[index];
+      
       const modifiedAttributes = modifiedStyleAttributesBuilder(
         docxDocumentInstance,
         vNode,
         attributes
       );
-      const tempRunFragments = await buildRun(childVNode, modifiedAttributes, docxDocumentInstance);
+      const passedAttributes = isVNode(childVNode) && childVNode.tagName === 'img'
```

Comment:
Why are we making a new variable here instead of modifying the modified attributes?

Reply:
Hey @nicolasiscoding 
A new variable is being made here because I cannot simply modify the passedAttributes.
We can skip this and directly use the ternary condition if needed